src/main/java/io/aiven/kafka/connect/common/output/jsonwriter/OutputFieldJsonWriter.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.output.jsonwriter;
+
+import java.io.IOException;
+
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+
+public interface OutputFieldJsonWriter {
+
+    JsonNode write(SinkRecord record) throws IOException;
```

Comment:
IMO we should rename the method to `build` or `create` or pass an JSON node for the field inside like `write(record, node)`

Reply:
I will rename it to build, but also, would rename it to Builder, to not add a confusion with a OutputFieldPlainWriter.